metadata.json

```diff
@@ -10,43 +10,43 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 <5.0.0"
+      "version_requirement": ">= 4.13.1 <10.0.0"
```

Comment:
You could use this breaking change to bump up the lowest level of stdlib you support.